migtests/tests/analyze-schema/run-analyze-schema-test

```diff
@@ -22,8 +22,14 @@ main() {
 	export version=$(yb-voyager version | awk -F'=' '/VERSION/ {print $2}')
 	sqlite3 ${EXPORT_DIR}/metainfo/meta.db "UPDATE json_objects SET json_text = json_set(json_text, '$.VoyagerVersion', '$version') WHERE key='migration_status';"
 
-	step "Run analyze schema"
-	analyze_schema --output-format json
+	step "Run analyze schema with default --output-format"
+	analyze_schema
+
+	step "Run analyze schema with --output-format xml"
```

Comment:
Should we also add validation to check whether these specific reports (XML, txt) exist because we only read the json in validations.

Reply:
done

Reply:
done, @priyanshi-yb